internal/workflow_shadower.go

```diff
@@ -338,10 +338,9 @@ func (o *ShadowOptions) validateAndPopulateFields() error {
 			}
 			statuses = append(statuses, status)
 		}
-		//All the open statuses are taken by default. This list seems to not work as expected.
-		//TODO: verify that the status list works as expected. currently all wfs of all types get picked up.
+		//This filter doesn't seem to be working as expected.
 		if len(statuses) == 0 {
-			statuses = []WorkflowStatus{WorkflowStatusOpen}
+			statuses = []WorkflowStatus{WorkflowStatusOpen, WorkflowStatusClosed}
 		}
 		queryBuilder.WorkflowStatus(statuses)
```

Comment:
I guess modern DBs with good query analyzers will just drop this expression, but evaluation of this does not make any sense.
I suggest making this optional:
```
if len(statuses) != 0 {
  queryBuilder.WorkflowStatus(statuses)
}
```

Reply:
will incorporate this and update PR.